charts/common/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: library
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.24
+version: 0.1.25
```

Comment:
In my humble opinion, we don't need to change the version of the Common chart templates for two reasons. First, in this case, no changes have been made to them. Second, in Folio Helm chart v.2, we do not use the Common chart templates; instead, we use the Folio-common chart templates.

Reply:
Agreed this change doesn't make any sense